waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java

```diff
@@ -31,12 +31,15 @@
 import org.springframework.stereotype.Component;
 
 import com.google.common.annotations.VisibleForTesting;
-
 import com.hotels.bdp.waggledance.conf.WaggleDanceConfiguration;
 
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
 @Component
 public class TTransportMonitor {
 
+  static final String METRIC_NAME_OPEN_TRANSPORTS = "open_transports_gauge";
```

Comment:
In this class you have a metric base name - (https://github.com/ExpediaGroup/waggle-dance/blob/main/waggle-dance-extensions/src/main/java/com/hotels/bdp/waggledance/extensions/client/ratelimit/RateLimitMetrics.java#L24)

Are you gonna add a base metric name also here to align with all metrics?

Reply:
yeah I should I was testing out how it looks with the prefix, but I'll add a namespace

Reply:
done